pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -868,10 +868,17 @@ export class CreateVmAction extends React.Component {
         if (this.props.systemInfo.osInfoList == null)
             return null;
 
+        let testdata;
+        if (!this.props.systemInfo.osInfoList)
+            testdata = "disabledOsInfo";
+        else if (!this.state.virtInstallAvailable)
+            testdata = "disabledVirtInstall";
+        else if (this.state.downloadOSSupported === undefined)
+            testdata = "disabledDownloadOS";
         let createButton = (
-            <Button disabled={!(this.props.systemInfo.osInfoList &&
```

Comment:
I don't see why the condition was incorrect.
It was disabled = !(osInfoListExists && virtInstallAvailable != undefined && (virtinstallAvailable != undefined && downloadFeatureSupported != udnefined))
In the test case bellow the virtInstallAvailable will be false,
so 
```
disabled = !(something && false && something_else) => disabled = !(false) => disabled = true;
```

Reply:
I think you incorrectly parsed the condition, as yoy replaced one || with &&, if virtInstallAvailable will be false, it will look like:
```
disabled = !(something && (false || true)) => disabled = !(true) => disabled = fase;
￼Write
```

Reply:
Well I think it's important to pass information of what's the reason why the button is disabled.
So far tests were passing, because the button was disabled, but for another reason as we expected. So we didn't notice that disabling button didn't work even thought we had a test for that.

Reply:
so let's pass the information - I don't disagree - but let's make sure to check that it's also disabled.

Reply:
Because it's the 'disabled' attribute that is what will block the user from clicking it